ansible/roles/azure/tasks/create_vmss.yml

```diff
@@ -214,18 +214,49 @@
     path: "{{ deploy_path }}/conf/muchos.props"
     line: "{{ item }} = namenode,resourcemanager,accumulomaster,zookeeper"
   with_items: "{{ instances_dict | json_query('[0].value') }}"
+  when: not hdfs_ha|bool
 
 - name: Assign metrics to the second node of the cluster
   lineinfile:
     path: "{{ deploy_path }}/conf/muchos.props"
     line: "{{ item }} = metrics"
   with_items: "{{ instances_dict | json_query('[1].value') }}"
+  when: not hdfs_ha|bool
 
 - name: Add worker nodes to muchos.props
   lineinfile:
     path: "{{ deploy_path }}/conf/muchos.props"
     line: "{{ item }} = worker"
   with_items: "{{ instances_dict | json_query('[2:].value') }}"
+  when: not hdfs_ha|bool
+
+- name: Assign Accumulo master, HDFS HA components cluster roles to the first node of the cluster
+  lineinfile:
+    path: "{{ deploy_path }}/conf/muchos.props"
+    line: "{{ item }} = namenode,resourcemanager,accumulomaster,zookeeper,journalnode,zkfc"
+  with_items: "{{ instances_dict | json_query('[0].value') }}"
+  when: hdfs_ha|bool
+
+- name: Assign Accumulo master, HDFS HA components cluster roles to the second node of the cluster
+  lineinfile:
+    path: "{{ deploy_path }}/conf/muchos.props"
+    line: "{{ item }} = zookeeper,metrics,journalnode,namenode,zkfc,accumulomaster"
+  with_items: "{{ instances_dict | json_query('[1].value') }}"
+  when: hdfs_ha|bool
+
+- name: Assign HDFS HA components cluster roles to the third node of the cluster
+  lineinfile:
+    path: "{{ deploy_path }}/conf/muchos.props"
+    line: "{{ item }} = journalnode,zookeeper"
+  with_items: "{{ instances_dict | json_query('[2].value') }}"
+  when: hdfs_ha|bool
+
+- name: Add worker nodes to muchos.props
+  lineinfile:
+    path: "{{ deploy_path }}/conf/muchos.props"
+    line: "{{ item }} = worker"
+  with_items: "{{ instances_dict | json_query('[3:].value') }}"
```

Comment:
In `muchos.props` for `azure.numnodes` there is comment saying it minimum should be >= 3.  Does HA impact that comment?

Reply:
Thanks for checking. Yes, you're spot on! I can update the comment to something like below. Let me know if this is ok?

```
# The minimum allowed size for this is 3 nodes for non-HA & 4 nodes for HA setup
```

Reply:
That sounds good.

Reply:
Addressed this on the latest commit